app/views/payments/index.html.erb

```diff
@@ -0,0 +1,35 @@
+<div class="category-header">
+   <a href="<%= categories_path %>">
+    <i class="fa-solid fa-arrow-left"></i>
+  </a>
+    <h2>My Transactions</h2>
+    <a href="/logout">
+        <i class="fa-solid fa-right-from-bracket"></i>
+    </a>
+</div>
+
+
+<div id="payments">
+
+<% if @payments.empty? %>
+      <p class="no-data">No transactions here yet.</p>
+    <% else %>
+  <% @payments.each do |payment| %>
+  <div class="single-payment">
+      <p class="payment-name"><%= payment.name %></p>
+      <div class="flex">
+        <p>Price: $ <%= payment.amount %> </p>
+        <%# <i class=" flex1 fa-solid fa-trash"></i> %>
+      </div>
+    </div>
+  <% end %>
```

Comment:
- [ ] Kindly ensure that your design matches the given design, currently, the transactions and the categories pages background color are different, let's fix this.

![design](https://github.com/atok624/Budget-App/assets/53124548/d27cee39-872d-4f0d-83c4-d7996c3b5907)


- [ ] Great job adding this page to list all the transactions, unfortunately, the total amount of transactions should be calculated and recorded at the top.

![total-tr](https://github.com/atok624/Budget-App/assets/53124548/366bb5bb-04c2-40af-837f-13a977588e63)